Description: Fix compilation with GCC6 by making implict conversion to bool explicit
Author: Axel Beckert <abe@debian.org>
Bug-Debian: https://bugs.debian.org/811660

--- a/libevolvotron/mutatable_image_computer.h
+++ b/libevolvotron/mutatable_image_computer.h
@@ -193,7 +193,7 @@
   //! Indicate whether computation us taking place (only intended for counting outstanding threads).
   bool active() const
     {
-      return _task;
+      return (bool)_task;
     }
 };
 
